### PR TITLE
docs: add user guide for GNOME's screenshot application

### PR DESCRIPTION
**Pull request by CBID2**
#### Description
For my final RSOD contribution,  I wrote a user guide for GNOME's Screenshot application

> [!NOTE]
> I also revised the installation section of the Decibels user guide 

#### Issue 

Relates to #2161 

#### Author checklist (Completed by original Author)
- [x] Good fit for the Rocky Linux project? Title and Author Metatags inserted ?
- [x] If applicable, steps and instructions have been tested to work
- [x] Initial self-review to fix basic typos and grammar completed

#### Rocky Documentation checklist (Completed by Rocky team) 
- [x] 1st Pass (Document is good fit for project and author checklist completed)
- [x] 2nd Pass (Technical Review - check for technical correctness) 
- [x] 3rd Pass (Detailed Editorial Review and Peer Review)
- [x] Final approval (Final Review)

### CODE REVIEW

File: docs/desktop/gnome/screenshot.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+﻿---
+title: Screenshot
+author: Christine Belzie
+contributors: 
+---
```
Comment (CBID2) [inline docs/desktop/gnome/screenshot.md:5]:
```suggestion

---
title: Screenshot 
author: Christine Belzie
contributors:
---

```

**Review by CBID2 — COMMENTED** [review]
fix metadata

**Comment by CBID2** [comment]
Hi @sspencerwire. Can you review and merge this please?

**Review by sspencerwire — APPROVED** [review]

**Review by sspencerwire — DISMISSED** [review]

File: docs/desktop/gnome/screenshot.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+3. Once you're satisfied with the screenshot, rename the file, and click on **Save**.
+
+   !!! tip
+
+    Want to save your computer's storage? Click on **Copy** to quickly add your screenshot to your work. 
+
+## Conclusion
+
+Whether it’s a picture for your online collage, computer's background, or a tutorial that you are creating for your favorite product, GNOME's Screenshot is the tool help you get the screenshots you want. Eager to learn more about or have more ideas for this application? [Submit an issue Screenshot repository at GitLab](https://gitlab.gnome.org/gnumdk/screenshot/-/issues).
```
Comment (CBID2) [inline docs/desktop/gnome/screenshot.md:40]:
```suggestion
Whether it’s a picture for your online collage, computer's background, or a tutorial that you are creating for your favorite product, GNOME's Screenshot is the tool help you get the screenshots you want. Eager to learn more about or have more ideas for this application? [Submit an issue to Screenshot’s repository at GitLab](https://gitlab.gnome.org/gnumdk/screenshot/-/issues).
```

**Review by CBID2 — COMMENTED** [review]
fix

**Comment by CBID2** [comment]
Can you approve it again @sspencerwire?

**Review by sspencerwire — APPROVED** [review]

**Pull request merged**